src/components/new-note-component/NewNote.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import { addNote } from "../../utils/notes-util/addNote";
+import { useNotes } from "../../context/useNotes";
+import "./NewNote.css";
+
+const NewNote = ({createNoteState:{createNote,setCreateNote}}) => {
+  const [tempNote, setTempNote] = useState({
+    _id: "",
```

Comment:
just a suggestion, make a variable for `initNoteState = { ... }`, then pass that into the initialState, makes it more readable

Reply:
Yes I will make this change